# HG changeset patch
# User Roman Neuhauser <neuhauser@sigpipe.cz>
# Date 1300350185 -3600
# Node ID bfd2d1e3f9fe2a43b95d42d2bcdae257c4e17457
# Parent  1857a674daf8a6b676ce32098da212dc8614e6b6
corrected manpage text for archive_write_set_bytes_per_block

diff --git a/libarchive/archive_write.3 b/libarchive/archive_write.3
--- a/libarchive/archive_write.3
+++ b/libarchive/archive_write.3
@@ -138,11 +138,7 @@
 Sets the block size used for writing the archive data.
 Every call to the write callback function, except possibly the last one, will
 use this value for the length.
-The third parameter is a boolean that specifies whether or not the final block
-written will be padded to the full block size.
-If it is zero, the last block will not be padded.
-If it is non-zero, padding will be added both before and after compression.
-The default is to use a block size of 10240 bytes and to pad the last block.
+The default is to use a block size of 10240 bytes.
 Note that a block size of zero will suppress internal blocking
 and cause writes to be sent directly to the write callback as they occur.
 .It Fn archive_write_get_bytes_per_block
